app/models/concerns/import_utils.rb

```diff
@@ -81,7 +81,7 @@ def person(arg)
       end
     end
 
-    {name: name, email: email}
+    {name: name, email: email}.with_indifferent_access
```

Comment:
Found two of these places, where the hash assembled in the importer used symbols as keys.  But the `author=` method on the model was assuming string keys.  This fixes setting the itunes-author name.